src/blocks/recommendations/README.md

```diff
@@ -1,44 +1,63 @@
 # Parse.ly Recommendations Block
 
-**IMPORTANT:** The Recommendations Block is available only in WordPress 5.9 and later.
+The Recommendations Block is designed to showcase links to content on your site as provided by the [Parse.ly `/related` API endpoint](https://www.parse.ly/help/api/recommendations#get-related). You can add it in Posts, Pages, or nearly any other custom post type. The Block can also be used in Full Site Editing (FSE) mode or as a [Block-based Widget](https://wordpress.org/support/article/block-based-widgets-editor/).
```

Comment:
Should we mention that this replaces the legacy widget?

Reply:
I've added a note about this.